swo/swomsg/log.go

```diff
@@ -31,7 +31,7 @@ func NewLog(ctx context.Context, db *sql.DB) (*Log, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer stdlib.ReleaseConn(db, conn)
+	defer func(db *sql.DB, conn *pgx.Conn) { _ = stdlib.ReleaseConn(db, conn) }(db, conn)
```

Comment:
Make a helper func to reduce noise and update the other two to take DB & Conn as well (they currently take Log & Conn)

Reply:
done